app/controllers/api/v1/portfolios_controller.rb

```diff
@@ -53,6 +53,7 @@ def restore
 
       def share
         portfolio = Portfolio.find(params.require(:portfolio_id))
+        authorize(portfolio, :share_or_unshare?)
```

Comment:
@eclarizio can we keep the actions as share? instead of share_or_unshare? and just use an alias method in the portfolio_policy so that its the same code. From the outside the action names could match the endpoint

Reply:
Sure, that's fine.